lists/ir.csv

```diff
@@ -798,3 +798,5 @@ https://www.tribunezamaneh.com/,NEWS,News Media,2020-01-30,ZamanehMedia,
 https://www.daadkhast.org/,NEWS,News Media,2020-01-30,ZamanehMedia,
 https://www.bonbast.com/,ECON,Economics,2020-06-22,Chelchela,blocked in Iran
 https://irandarkhamooshi.net/fa,HACK,Hacking Tools,2020-11-05,Chelchela,
+https://paskoocheh.com/,ANON,Anonymizers and censorship circumvention,2021-07-18,
+https://www.toosheh.org/,ANON,Anonymizers and censorship circumvention,2021-07-18,
```

Comment:
The description of the category code is mistyped. It should read `Anonymization and circumvention tools`. It's also missing the author column